crates/smartmodule-development-kit/src/clean.rs

```diff
@@ -0,0 +1,26 @@
+use std::fmt::Debug;
+
+use anyhow::Result;
+use clap::Parser;
+
+use cargo_builder::cargo::Cargo;
+
+/// Calls cargo to clean the current working directory.
+#[derive(Debug, Parser)]
+pub struct CleanCmd {
+    /// Extra arguments to be passed to cargo
+    #[arg(raw = true)]
+    extra_arguments: Vec<String>,
+}
+
+impl CleanCmd {
+    pub(crate) fn process(self) -> Result<()> {
+        println!("{:?}", self.extra_arguments);
```

Comment:
I think this can be removed!

Reply:
That horrible println is removed, sorry. I will create the test as soon as I can free up some time. Thank for your help so far.

Reply:
No worries! Thanks so much for taking the time!

Reply:
Hey @EstebanBorai. I am facing issues running the CLI tests locally. I have no previous experience with bats. Do you have any clue? 
![image](https://github.com/user-attachments/assets/10d853d1-ff85-4cf9-83c7-8672ffcf2fa4)

Reply:
NVM. It is working now. I just did 'fluvio cluster start' and the file missing file was created.